x/pricefeed/simulation/operations/msg.go

```diff
@@ -101,26 +107,36 @@ func getExpiryTime(ctx sdk.Context) (t time.Time) {
 // pickNewRandomPrice picks a new random price given the current price
 // It takes the current price then generates a random number to multiply it by to create variation while
 // still being in the similar range. Random walk style.
```

Comment:
nit: these are a bit out of date given the new changes

In general I'm liking the comments, but it's worth remembering they can get out of sync as code changes (or is copy and pasted to other areas), which can lead to misleading comments.

The general advice is to make sure comments add useful information not obvious from the code itself. For example something like this wouldn't add anything:
```go
    // get next time by adding period to current time
    nextTime := currentTime.Add(period)
```

Reply:
🆗